parsl/tests/test_radio/diaspora_login.py

```diff
@@ -0,0 +1,6 @@
+from diaspora_event_sdk import Client as GlobusClient
```

Comment:
What is this diaspora_login.py for? Can you add some usage notes about when it might be used?

Reply:
What service is this logging into? / What kind of account is needed? Github Actions (which runs our automated testing) supports being logged into services (for example, that is how automated release publication works to PyPI) so possibly that is an approach to getting the test running in CI?

Reply:
Hi Ben, this is Haochen, I am a GLabs student working on the diaspora project. The log goes to an AWS-managed Kafka cluster maintained by the diaspora team. The login process works similarly to that of the Compute SDK, so any accounts supported by Globus Auth can log in. I'll discuss with Ryan this week to figure out how to log in during CI and how to drop the boto3 dependency. Thank you!

Reply:
Do you think I shall mock the behaviors related to diaspora in order to run the testable parts instead of skipping the whole thing?

Reply:
Hi Ben, 

Ryan and I agree that Parsl's CI should not depend on the diaspora Kafka cluster status, so mock sending to Diaspora in tests may be needed in the future. Also, after a recent paper deadline, we will refactor the indirect boto3 dependency in the Diaspora SDK to remove the `botocore==1.29.125` constraint in Parsl's test.

Do you think Claudia can keep the diaspora test optional for this PR and add mock sending tests in a future PR?

Reply:
I don't see that there's an urgent need to rush this PR into `master` without tests and with awkward package dependencies. Hopefully your potential users will be ok to install from a branch at this prototype stage until you are able to work on the PR some more after your more important targets.

It's fine to leave this PR open until you have more time to work on it. You can convert it to a draft if you would like to indicate that status - I do that often for my own PRs - there's a link "Convert to draft" in the top right corner of the Github web interface.